src/main/java/model/GetAllUserProfileDetails.java

```diff
@@ -0,0 +1,14 @@
+package model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+
+@NoArgsConstructor
+@Data
+@AllArgsConstructor
+public class GetAllUserProfileDetails {
```

Comment:
This should be named GetUserProfileResponse should be similar to the lambda name (GetUserProfileLambda)